nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -804,6 +827,33 @@ where
             .all(|blob| sampled_blobs_ids.contains(&blob.blob_id()));
         validated_blobs
     }
+
+    fn log_received_block(
+        received_blocks_counter: &Counter<u64>,
+        received_blocks_data_histogram: &Histogram<u64>,
+        block: &Block<ClPool::Item, DaPool::Item>,
+    ) {
+        let content_size = block.header().content_size();
+        let transactions = Value::I64(block.cl_transactions_len() as i64);
+        let blobs = Value::I64(block.bl_blobs_len() as i64);
```

Comment:
How could I handle data truncation here?
Both methods return a `usize` but largest data type for `Value` is `I64`. That or `String`.